src/components/header/Header.js

```diff
@@ -1,7 +1,8 @@
 import React from "react";
```

Comment:
This import line needs to be removed as component is no longer functional.

Reply:
No, wrong file. Import line needed to be removed from HeaderOption.js, which was done in second commit